src/flexicarousel.es6.js

```diff
@@ -170,6 +170,30 @@ export default class Carousel {
 		this.current = to;
 	}
 
+	// ------------------------------------- Event Listeners ------------------------------------- //
+
+	_createHandleBindings() {
+
+		return {
+			'touchstart': this._dragStart.bind(this),
```

Comment:
BTW - because we're using .bind(this) to attach the eventListener, we cannot actually ever remove it -- there is no reference to the function (technical aside: bind create an anonymous closure).  So, long story short, destroy() doesn't do anything and memory leaks abound.